matrix-sdk/src/main/java/org/matrix/androidsdk/MXSession.java

```diff
@@ -2076,6 +2076,45 @@ public void onUnexpectedError(Exception e) {
         });
     }
 
+    /**
+     * Deactivate the account.
+     *
+     * @param context       the application context
+     * @param type          type of authentication
+     * @param userId        current user id
```

Comment:
A MXSession is tied to a user. The userId should not appear in arguments. `mCredentials.userId` can be used instead.